solidity/contracts/external/IMedianizer.sol

```diff
@@ -5,12 +5,12 @@ pragma solidity 0.5.17;
 interface IMedianizer {
     /// @notice Get the current price.
     /// @dev May revert if caller not whitelisted.
-    /// @return Price (USD) with 18 decimal places.
+    /// @return Designated price with 18 decimal places.
     function read() external view returns (uint256);
 
     /// @notice Get the current price and check if the price feed is active
     /// @dev May revert if caller not whitelisted.
     /// @return Price (USD) with 18 decimal places.
```

Comment:
`Price (USD) -> Designated price` like above?
` with 18 decimal places` is tris true for all maker medianizers?

Reply:
It appears to be true for all of them, yes.